hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/capacity/CapacitySchedulerConfiguration.java

```diff
@@ -723,6 +727,17 @@ public <S extends SchedulableEntity> OrderingPolicy<S> getAppOrderingPolicy(
     return orderingPolicy;
   }
 
+  public void setOrderingPolicy(String queue,
+                 String appOrderingPolicy, String postfix, boolean value) {
+    setBoolean(getQueuePrefix(queue) +
+        "ordering-policy" + DOT + appOrderingPolicy + DOT + postfix, value);
+  }
+
+  public boolean getOrderingPolicy(String queue, String appOrderingPolicy, String postfix) {
+    return getBoolean(getQueuePrefix(queue) +
+        "ordering-policy" + DOT + appOrderingPolicy + DOT + postfix, false);
+  }
```

Comment:
This naming is really misleading, as it doesn't get the ordering policy, but it gets the properties of that policy. Since the ordering policy is pluggable, anyone can write one I think we shouldn't introduce getters for it, especially not fixed return type ones.

Reply:
Yes, the naming is misleading(for now I have removed this getter and setter), if we don't introduce a getter and setters for this then the purpose of this the jira seems to be incomplete. So what do you suggest, do we go without this getter and setter for the above property.?